homeassistant/components/islamic_prayer_times/coordinator.py

```diff
@@ -145,9 +146,12 @@ async def _async_update_data(self) -> dict[str, datetime]:
             async_call_later(self.hass, 60, self.async_request_update)
             raise UpdateFailed from err
 
+        # introduced in prayer-times-calculator 0.0.8
+        prayer_times.pop("date", None)
```

Comment:
Why do we need this pop?

Reply:
Version 0.0.8 of the library added a 'date' key on the same level as the prayer times key. We are not using this value in this integration and if we loop over this key it will raise an error when parsing time.